src/ucp/api/ucp.h

```diff
@@ -357,6 +357,11 @@ typedef struct ucp_params {
      * This field defaults to 0 if not specified.
      */
     uint64_t                           tag_sender_mask;
+    /**
+     * Thread access mode to the context.
+     * This field defaults to UCS_THREAD_MODE_SINGLE if not specified.
+     */
+    ucs_thread_mode_t                  thread_mode;
```

Comment:
No, lets keep it as is. I don't want to have duplicate definitions/types. AFAIK only reason is doxygen not able to recognize the type, but i'm sure we can find a better solution for this (like, for example, add ucs header files to doxygen)